src/server.ts

```diff
@@ -1,18 +1,21 @@
 import './config/load-env'; // Load our environment variables
 
 import * as treehouse from 'tree-house';
-import { logger, raven } from './lib/logger';
+import { logger, sentry } from './lib/logger';
 import { importTranslations } from './lib/translator';
 import { app } from './app';
 import { errorTranslations } from './constants';
 
-raven.config(process.env.SENTRY_DSN, {
-  environment: process.env.NODE_ENV,
-}).install();
+sentry.init({ dsn: process.env.SENTRY_DSN });
```

Comment:
How does Sentry make the difference between environments now? With the same `SENTRY_DSN` as before?

Reply:
Sentry.init({
  environment: 'staging',
})

https://docs.sentry.io/enriching-error-data/environments/?platform=node

Adding environment config to sentry